lib/src/google_api_availability.dart

```diff
@@ -40,4 +44,100 @@ class GoogleApiAvailability {
 
     return GooglePlayServicesAvailability.values[availability];
   }
+
+  /// Returns true if the device was able to set Google Play Services to available,
+  /// Returns false if the device was unable to set Google Play Services to available or status is unknown.
+  ///
+  /// If it is necessary to display UI in order to complete this request
+  /// (e.g. sending the user to the Google Play store) the passed Activity will be used to display this UI.
```

Comment:
The documentation for this (and other methods) could be improved. Try to describe what the method does instead of only the return value. 

To make it easier on yourself, it is perfectly fine to copy the documentation from the official Android documentation. Just make sure you remove the part about the `activity` as from Dart the user doesn't have control over it so it doesn't make sense to name it. 

An example on how I would do it is:
```suggestion
    /// Attempts to make Google Play services available on this device. 
    ///
    /// If the `Future` completes without throwing an exception, Play Services 
    /// is available on this device.
```